### PR TITLE
Form

### PR DESCRIPTION
**Pull request by jk0ehn**

### CODE REVIEW

File: src/pages/contact.js [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+	background-color: #61adad;
+	color: white;
+`;
+
+export default function Contact() {
+	const handleSubmit = event => {
+		event.preventDefault();
+		const formdata = new FormData(event.target);
+		const data = Object.fromEntries(formdata);
+		console.log(data);
```
Comment (K-Juno) [inline src/pages/contact.js:36]: Do you need this console.log?

File: src/pages/_document.js [hunk 2]
```diff
@@ -1,20 +1,88 @@
...
+
+import createEmotionCache from '../config/emotion';
+import theme from '../config/theme';
+
+export default class MyDocument extends Document {
+	render() {
+		return (
+			<Html lang="en">
+				<Head>
+					{/* PWA primary color */}
```
Comment (K-Juno) [inline src/pages/_document.js]: There's a comment there.

File: src/pages/_document.js [hunk 3]
```diff
@@ -1,20 +1,88 @@
...
+				<body>
+					<Main />
+					<NextScript />
+				</body>
+			</Html>
+		);
+	}
 }
+
+// `getInitialProps` belongs to `_document` (instead of `_app`),
```
Comment (K-Juno) [inline src/pages/_document.js]: You have a lot of comments in general, just FYI.

**Review by K-Juno — COMMENTED** [review]
Hi, just noticed a console.log or two and still lots of comments, not sure whether they're intentional.

File: src/pages/contact.js (hunk 1 above)
Comment (jk0ehn) [reply src/pages/contact.js:36]: Thanks for review my code - The console.log is only for testing till I finished my feature. So I pushed my pull-request that it is not to long for review.

**Pull request merged**